docs/nuctl/nuctl_build.md

```diff
@@ -17,7 +17,7 @@ nuctl build function-name [options] [flags]
   -h, --help                    help for build
   -i, --image string            Container image to use, will use function name if not specified
       --nuclio-src-dir string   Local directory with nuclio sources (avoid cloning)
-      --nuclio-src-url string   nuclio sources url for git clone (default "git@github.com:nuclio/nuclio.git")
+      --nuclio-src-url string   nuclio sources url for git clone (default "https://github.com/nuclio/nuclio.git")
```

Comment:
For the future, maybe we generate the documentation automatically (at least this part) by running `nuctl --help`

Reply:
Kubernetes has stopped checking in automatically generated help files due to this exact reason:

https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/kubectl/kubectl.md